+++ Makefile.am	2009-09-15 13:07:19.000000000 -0300
--- Makefile.am	2009-04-17 17:27:57.000000000 -0300
@@ -74,9 +74,7 @@
 NTOPDATA = ntop-cert.pem \
            $(ETTER_PASSIVE) \
            oui.txt.gz \
-           specialMAC.txt.gz \
-	   GeoIPASNum.dat \
-	   GeoLiteCity.dat
+           specialMAC.txt.gz
 
 NTOPHTML = html html/*.js html/*.html  html/*.gif html/*.jpg html/*.ico html/*.png \
            html/*.css html/*.dtd \
