--- configure.in.orig	2011-05-16 19:54:07.000000000 +0200
+++ configure.in	2011-05-16 19:54:28.000000000 +0200
@@ -1810,31 +1810,6 @@
 if test ".${ac_cv_lib_GeoIP_GeoIP_record_by_ipnum}" = ".yes"; then
 LIBS="$LIBS -lGeoIP"
 
-if test -f "GeoLiteCity.dat"; then
-   echo "GeoLiteCity.dat already present"
-else
-   if test -f "3rd_party/GeoLiteCity.dat.gz"; then
-     cp 3rd_party/GeoLiteCity.dat.gz .
-   else
-     wget http://www.maxmind.com/download/geoip/database/GeoLiteCity.dat.gz
-   fi
-
-   gunzip GeoLiteCity.dat.gz
-fi
-
-dnl> --------
-
-if test -f "GeoIPASNum.dat"; then
-   echo "GeoIPASNum.dat already present"
-else
-   if test -f "3rd_party/GeoIPASNum.dat.gz"; then
-     cp 3rd_party/GeoIPASNum.dat.gz .
-   else
-     wget http://www.maxmind.com/download/geoip/database/asnum/GeoIPASNum.dat.gz
-   fi
-
-   gunzip GeoIPASNum.dat.gz
-fi
 else
 AC_MSG_RESULT(Please install GeoIP (http://www.maxmind.com/))
 exit 1
